From: "Lucas B. Cohen" <lucas.cohen@free.fr>
Date: Thu, 10 Feb 2011 11:31:21 +0100
Subject: Typo in ignore Regex help

Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=129478
---
 strings.ml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/strings.ml b/strings.ml
index 36861d9..1275399 100644
--- a/strings.ml
+++ b/strings.ml
@@ -4011,7 +4011,7 @@ let docs =
       \032      changed. Instead of putting a line of the form\n\
       \032                ignore = <regexp>\n\
       \032      in your profile (.unison/default.prf), you should put:\n\
-      \032                ignore = Regexp <regexp>\n\
+      \032                ignore = Regex <regexp>\n\
       \032      Moreover, two other styles of pattern are also recognized:\n\
       \032                ignore = Name <name>\n\
       \032      matches any path in which one component matches <name>, while\n\
-- 
