From: "Lucas B. Cohen" <lucas.cohen@free.fr>
Date: Thu, 10 Feb 2011 11:35:39 +0100
Subject: Typo: missing o in follow

Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=466173
---
 strings.ml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/strings.ml b/strings.ml
index b856170..0976703 100644
--- a/strings.ml
+++ b/strings.ml
@@ -425,7 +425,7 @@ let docs =
       \n\
       \032  indicates that the file c has been modified only in the second\n\
       \032  replica, and that the default action is therefore to propagate the new\n\
-      \032  version to the first replica. To follw Unison's recommendation, press\n\
+      \032  version to the first replica. To follow Unison's recommendation, press\n\
       \032  the \"f\" at the prompt.\n\
       \n\
       \032  If both replicas are modified and their contents are different, then\n\
-- 
