From: Sylvain Le Gall <gildor@debian.org>
Date: Thu, 10 Feb 2011 11:37:46 +0100
Subject: Fix -silent behaviour

Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=497729
---
 uitext.ml |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/uitext.ml b/uitext.ml
index d00fd7e..cc6a839 100644
--- a/uitext.ml
+++ b/uitext.ml
@@ -745,7 +745,8 @@ let start _ =
     Uicommon.uiInit
       (fun s -> Util.msg "%s\n%s\n" Uicommon.shortUsageMsg s; exit 1)
       (fun s -> Util.msg "%s" Uicommon.shortUsageMsg; exit 1)
-      (fun () -> if not (Prefs.read silent)
+      (fun () -> if Prefs.read silent then Prefs.set Trace.terse true;
+                 if not (Prefs.read silent)
                  then Util.msg "%s\n" (Uicommon.contactingServerMsg())) 
       (fun () -> Some "default")
       (fun () -> Util.msg "%s" Uicommon.shortUsageMsg; exit 1)
-- 
